### ISSUE TITLE
gdeploy: IdentityGroupFilter is being ignored

### ISSUE THREAD

**Issue by keymon** [common-fate/glide#621]
# What?

The setting `IdentityGroupFilter` from `deployment.yaml` documented here: https://docs.commonfate.io/common-fate/sso/filtering-user-group-import has no effect.

# how to reproduce

Given a deployment like the one below, set `IdentityGroupFilter` and rerun `gdeploy update`. Nothing happens. 

```diff
version: 2
deployment:
  stackName: commonfate-non-prod
  account: "1234567890"
  region: us-west-2
  release: v0.15.6
  parameters:
    CognitoDomainPrefix: common-fate-non-prod-login-mycompany
    AdministratorGroupID: vhjdsfbkgbsakdng1
    IdentityProviderType: okta
    SamlSSOMetadataURL: https://mycompany-test.oktapreview.com/app/exka4ccnoe1AZdYy91d7/sso/saml/metadata
+    IdentityGroupFilter: "Okta-App.*Commonfate|commonfate_example.*"
    ProviderConfiguration:
      okta:
        uses: commonfate/okta@v1
        with:
          apiToken: awsssm:///granted/providers/okta/apiToken:1
          orgUrl: https://mycompany-test.oktapreview.com
    IdentityConfiguration:
      okta:
        apiToken: awsssm:///granted/secrets/identity/okta/token:4
        orgUrl: https://mycompany-test.oktapreview.com/
```

# Tested on 

gdeploy 0.15.6

**Comment by keymon** [comment]
Possible fix https://github.com/common-fate/common-fate/pull/622

**Comment by keymon** [comment]
Solved by #622
